src/socket.rs

```diff
@@ -861,6 +861,41 @@ impl Socket {
         }
     }
 
+    /// IP_TRANSPARENT (since Linux 2.6.24)
```

Comment:
Could you following the convention of the other settings. I.e. Set value for the `IP_TRANSPARENT` option on this socket.

Reply:
Missed this one with last commit.  Do you want me to remove the text that's there and simply replace it with what you suggested?  Or add that as a header?